src/app/features/reading-now/reading-now.component.ts

```diff
@@ -0,0 +1,30 @@
+import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
+
+import { BookCardComponent } from './book-card.component';
+import { CommonModule } from '@angular/common';
+import { BooksService } from '../../services/books.service';
+import { Book } from '../../models/book.model';
+
+@Component({
+  selector: 'app-reading-now',
+  standalone: true,
+  templateUrl: './reading-now.component.html',
+  styleUrls: ['./reading-now.component.css'],
+  imports: [BookCardComponent, CommonModule],
+})
+export class ReadingNowComponent implements OnInit {
+  public dataBooks: Book[] | null = null;
+  constructor(
+    private dataService: BooksService,
+    private cd: ChangeDetectorRef
+  ) {}
+  ngOnInit(): void {
+    this.dataService.getData().subscribe(
+      (data) => {
+        this.dataBooks = data;
+        this.cd.detectChanges();
```

Comment:
This line looks a little weird. I suggest to modify the logic of the app following the next pattern:

https://blog.angular-university.io/angular-2-smart-components-vs-presentation-components-whats-the-difference-when-to-use-each-and-why/

https://medium.com/@mrahmedkhan019/smart-dumb-components-in-angular-e1d6dbd6edff

The main idea is to have presentation components with change detection strategy on push and smart components with the default change detection strategy. The smart components always will be the parent components.